pages/success-banner/index.js

```diff
@@ -1,23 +1,49 @@
 import React from 'react';
 import Head from 'next/head';
 
+/**
+ * @summary Make and return an event method for a given event description string
+ */
 const eventLog = (eventDesc) => {
   const data = 'Banner ' + eventDesc;
 
   return () => {
     const url = new URL(location.href);
 
-    if (url.searchParams.get('etcher-version').indexOf('1.0.0') !== -1) {
-      console.log(data);
-
-    } else {
+    // Use the 'new' API version for more recent versions of Etcher utilizing
+    // the Robot object format.
+    if (url.searchParams.get('api-version') === '1') {
```

Comment:
Since we don't know what way the API will change for `'2'` I keep it strict, and it's easy to change a banner, it doesn't have to be distributed to everyone for the changes to propagate.